packages/frontend/src/app/App.js

```diff
@@ -161,30 +162,21 @@ class Routing extends Component {
             { name: 'Українська', code: 'ua' },
         ];
 
-        const browserLanguage = getBrowserLocale(
-            languages.map((l) => l.code)
-        );
+        const browserLanguage = getBrowserLocale(languages.map((l) => l.code));
```

Comment:
This is what I was talking about the other day- that we're getting a lot of formatting changes now... Maybe because formatting is only happening on "staged" files (I think there's `lint-staged` running things in the repo)

Reply:
I think we might need to create a PR just to format everything, and then we do the next changes on top of that. Would make things a lot easier to review.

Reply:
Agree, will prepare a PR after this is merged.